### PR TITLE
Rock 3D: Version 1.000 added

### PR DESCRIPTION
**Pull request by aaronbell**
d3bb48e: [gftools-packager] Rock 3D Font: Version 1.000 added

* Rock 3D Font Version 1.000 taken from the upstream repo https://github.com/shibuyafont/3d-rock-font.git at commit https://github.com/shibuyafont/3d-rock-font/commit/cf98f864221ccfe24091dccc23325b2d97a0f24b.

 d736ab9: [gftools-packager] ofl/rock3dfont remove METADATA "source".  google/fonts#2587

### CODE REVIEW

File: ofl/rock3dfont/DESCRIPTION.en_us.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<p>
```
Comment (m4rc1e) [inline ofl/rock3dfont/DESCRIPTION.en_us.html:1]: Imo we need better descriptions for these Shibuya fonts.

Comment (aaronbell) [reply ofl/rock3dfont/DESCRIPTION.en_us.html:1]: I've asked Min-Young to follow up and see if they can give us more font-specific descriptions.

**Comment by aaronbell** [comment]
Did we decide if the descriptions will stay as is? 
Also, should "font" be removed from this project name as well (like Palette Mosaic and Shizuru)?

**Comment by mintoming** [comment]
The designer never got back to us about the description, so let me check with him for both - description and the "font" in the name.

**Comment by mintoming** [comment]
Hi all, the designer agreed to omit "font" from all three fonts. 
Also, he updated more description for all three fonts too.
So, we're good to go! :)

**Comment by mintoming** [comment]
New description here:

This is a font based on the sketch of 3D letters by handicapped artists, refined & created by design major students, capturing the solidness of the 3D with two dimensional sketch. This is one of the work of Shibuya Font project, a collaboration of design major students and handicapped artist living in Shibuya city, officially approved by Shibuya city in Tokyo. 
http://www.shibuyafont.jp

**Comment by aaronbell** [comment]
@m4rc1e Ready for review

**Review by m4rc1e — CHANGES_REQUESTED** [review]
- "Font" needs to be removed from the family name
- Description is still empty so please add the text Min provided.

**Comment by aaronbell** [comment]
@m4rc1e Got it all updated and ready for you.

**Pull request merged**